lib/src/components/NxTab/NxTab.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { ActiveTabContext } from '../NxTabs/NxTabs';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxTab = function NxTabElement(props: Props) {
+  // Use React.useContext instead of importing useContext for jest to mock the value in the test
+  const activeTab = React.useContext(ActiveTabContext);
+  const {id, className, ...attrs} = props;
+  const active = activeTab === id;
+  const classNames = classnames('nx-tab', className, { active });
+  const selected = active ? 'true' : 'false';
+
+  return <li role="tab" id={id} aria-selected={selected} className={classNames} {...attrs}/>;
```

Comment:
This seems to work just as well:

```suggestion

  return <li role="tab" id={id} aria-selected={active} className={classNames} {...attrs}/>;
```

Reply:
That did not work in my testing. It sets the attribute to `aria-selected=""` for example.

Reply:
> That did not work in my testing. It sets the attribute to aria-selected="" for example.

Strange, I've just reconfirmed that it works for me, setting `aria-selected="false"` or `aria-selected="true"` in the DOM explorer

Reply:
Checked again, and it's working for me, so I removed the manual translation. Not sure why it didn't work before. 🤷